ibis/backends/tests/test_export.py

```diff
@@ -214,7 +213,9 @@ def test_table_to_parquet(tmp_path, backend, awards_players):
 
     df = pd.read_parquet(outparquet)
 
-    backend.assert_frame_equal(awards_players.to_pandas(), df)
+    backend.assert_frame_equal(
+        awards_players.to_pandas().fillna(pd.NA), df.fillna(pd.NA)
```

Comment:
Blaah.

Reply:
Phase 2 after the port?? 😂

Reply:
Probably, we should clean up the output coercions.